sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -223,6 +223,28 @@ object FileFormatWriter extends Logging {
       processStats(description.statsTrackers, ret.map(_.summary.stats))
       logInfo(s"Finished processing stats for write job ${description.uuid}.")
 
+      val insertStats : String = try {
+        description.statsTrackers.map{
+          s =>
+            val header = s"File stats [ "
+            val metrics = s.asInstanceOf[BasicWriteJobStatsTracker].metrics
+            val inner = metrics.map{
+              p => p._1 + "=" + p._2.value
```

Comment:
method + in class Int is deprecated since 2.13.0,  we should use the string interpolation `s"$num$str"`